README.md

```diff
@@ -64,20 +64,21 @@
         </tr>
     </thead>
     <tbody>
+        <!-- not published yet -->
         <tr>
             <td>
-                <a href="https://github.com/privacy-scaling-explorations/zk-kit.rust/tree/main/crates/example">
-                    example
+                <a href="https://github.com/privacy-scaling-explorations/zk-kit.rust/tree/main/crates/imt">
+                    imt
                 </a>
             </td>
             <td>
-                <a href="https://docs.rs/example">
-                    <img src="https://img.shields.io/crates/d/example?style=flat-square" alt="Crate version" />
+                <a href="https://privacy-scaling-explorations.github.io/zk-kit.rust/imt">
+                    <img src="https://img.shields.io/crates/v/zk-kit-rs-imt?style=flat-square" alt="Crate version" />
                 </a>
             </td>
             <td>
-                <a href="https://docs.rs/example">
-                    <img src="https://img.shields.io/crates/v/example?style=flat-square" alt="Crate downloads" />
+                <a href="https://privacy-scaling-explorations.github.io/zk-kit.rust/imt">
+                    <img src="https://img.shields.io/crates/d/zk-kit-rs-imt?style=flat-square" alt="Crate downloads" />
```

Comment:
@sripwoud is there anything similar to NPM scopes in Rust?

I don't think we need to specify "rs" 🤔

Reply:
on crates.io you can't have a scope/organization (`@scope`) in the pkg name like in npm.
But regarding ownership, it is possible to add a github team 
with `cargo owner --add github:org:team`
see [docs](https://doc.rust-lang.org/cargo/reference/publishing.html?highlight=orga#cargo-owner)

So should we create a zk-kit team on github?

also captured this issue to add a workflow to release the crates #10

Reply:
about the pkg name, should we go with `zk-kit-imt`?
And name all our crates `zk-kit-<pkg>`?

Reply:
renamed to `zk-kit-imt`

Reply:
Yes makes sense!